### DISCUSSION TITLE
`register` not registering field  with name has a single quote in the middle,

### DISCUSSION THREAD

**Discussion by Mallikarjunamatla** [react-hook-form/react-hook-form#7494] [Ideas] [answered]
I' m passing name with template literal's  as 
`
import React from "react";
import ReactDOM from "react-dom";
import { useForm } from "react-hook-form";

import "./styles.css";

export default function App() {
  const { register, handleSubmit } = useForm();
  const onSubmit = (data) => console.log(data);
const name = "Patient's Name";
  return (
    <form onSubmit={handleSubmit(onSubmit)}>
      <input {...register(`register ${name}`)} />
      <select {...register("gender")}>
        <option value="female">female</option>
        <option value="male">male</option>
        <option value="other">other</option>
      </select>
      <input type="submit" />
    </form>
  );
}

const rootElement = document.getElementById("root");
ReactDOM.render(<App />, rootElement);
`
Output : 
{register Patients Name: undefined, gender: "female"}

**Comment by bluebill1049** [comment, marked as answer]
please avoid special character and also see if you can provide a codesandbox next time with questions.

**Reply by Mallikarjunamatla** [reply]
Thank you, Here is the codesandbox for it : https://codesandbox.io/s/react-hook-form-js-forked-3rlw8?file=/src/App.js
